newIDE/app/src/ObjectEditor/ObjectExporterDialog.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+import { Trans } from '@lingui/macro';
+import React from 'react';
+import FlatButton from '../UI/FlatButton';
+import Dialog from '../UI/Dialog';
+import HelpButton from '../UI/HelpButton';
+import { Column, Line } from '../UI/Grid';
+import CloudUpload from '@material-ui/icons/CloudUpload';
+import { LineStackLayout } from '../UI/Layout';
+import RaisedButton from '../UI/RaisedButton';
+import Text from '../UI/Text';
+import EventsFunctionsExtensionsContext, {
+  type EventsFunctionsExtensionsState,
+} from '../EventsFunctionsExtensionsLoader/EventsFunctionsExtensionsContext';
+import Window from '../Utils/Window';
+
+const exportCustomObject = (
+  eventsFunctionsExtensionsState: EventsFunctionsExtensionsState,
+  customObject: gdObject
+) => {
+  const eventsFunctionsExtensionWriter = eventsFunctionsExtensionsState.getEventsFunctionsExtensionWriter();
+  if (!eventsFunctionsExtensionWriter)
+    return Promise.reject(new Error('Not supported'));
```

Comment:
Use an async function and do:
```suggestion
    throw new Error('Not supported');
```